plot/plot.app.js

```diff
@@ -683,14 +695,56 @@
+                                                                try{
+                                                                    traceLabel=ERMrest.renderMarkdown(ERMrest.renderHandlebarsTemplate(trace.legend_markdown_pattern,$rootScope.templateParams),true);
+                                                                    //Defined regex pattern to extract the url in markdown pattern
+                                                                    //Example : traceLabel: <a href="/chaise/recordset/#2/Gene_Expression:Specimen/*::facets::N4IghgdgJiBcDaoDOB7ArgJwMYFM6JHQBcAjdafEAQQiIEsyoBPEAGmtoZWYH0AVHEiJIeAYRQRUGemgC2PAGYBrHCwC6AX3YAlAJIAREGvZYAFijq4klAIoBaAMwBpAKwA2I1uTpseBKGIyNAoEDnpGFnYacO4mfkFhMUwwegkeAGUiFLQRZVVPHQMjE3NLQVs7AE03AEYAWU9NIA&pcid=plotApp&ppid=2oio1qej2i6m2bv62ox21h1q" target="_blank">Imaging: Histology</a>
+                                                                    // extractedLink = /chaise/recordset/#2/Gene_Expression:Specimen/*::facets::N4IghgdgJiBcDaoDOB7ArgJwMYFM6JHQBcAjdafEAQQiIEsyoBPEAGmtoZWYH0AVHEiJIeAYRQRUGemgC2PAGYBrHCwC6AX3YAlAJIAREGvZYAFijq4klAIoBaAMwBpAKwA2I1uTpseBKGIyNAoEDnpGFnYacO4mfkFhMUwwegkeAGUiFLQRZVVPHQMjE3NLQVs7AE03AEYAWU9NIA&pcid=plotApp&ppid=2oio1qej2i6m2bv62ox21h1q
+                                                                    var extractedLink=/<a\b.+href="([^\n\r]*)"\s/ig.exec(traceLabel)[1];
+                                                                    var linkWithContextParams =extractedLink+qCharacter+"pcid="+contextUrlParams.cid+"&ppid="+contextUrlParams.pid;
+                                                                    values.legend_markdown_pattern.push(linkWithContextParams);
+                                                                    traceLabel=traceLabel.replace(extractedLink,linkWithContextParams)
+                                                                }catch(error){
+                                                                    traceLabel=row[trace.legend_col];
+                                                                }
+                                                                //delete $rootScope.templateParams;
+                                                            }
+                                                            values.labels.push(traceLabel);
+                                                            //Added the legend column names to text variable so that the tooltip name does not contain a link if the legends contains a link
+                                                            var tooltip=trace.hovertemplate_display_pattern ? ERMrest.renderHandlebarsTemplate(trace.hovertemplate_display_pattern, $rootScope.templateParams) : row[trace.legend_col];
+                                                            values.text.push(tooltip);
```

Comment:
Did you test this with a `hovertemplate_display_pattern` defined? 

It doesn't have to be a `<a>` with a link. It could be `<strong>sample text</strong>` just to make sure the HTML element is applied properly.

Reply:
Future Scope